src/containers/SideNav/SideNav.test.js

```diff
@@ -142,7 +142,7 @@ it('SideNav redirects to root when all namespaces selected on namespaced URL', a
       <SideNav
         extensions={[]}
         history={{ push }}
-        match={{ params: { namespace } }}
+        match={{ params: { namespace }, url: '/secrets' }}
```

Comment:
use `urls` util instead of hardcoding URLs, including in tests
this makes it far easier to maintain and avoids unnecessary code change if we modify the URL structures

there are a few more below that need updating too